### ISSUE TITLE
[Bug] [Linux] Millennium Fails to start and spams logs

### ISSUE THREAD

**Issue by agedplato** [shdwmtr/millennium#172]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a theme
- [X] My issue is not caused by a plugin

### Describe the Bug

After running start.sh W/konsole steam opens but without millennium functionality. (I.E. themes, plugins, etc. in settings page.)

Output after running start.sh
[log.txt](https://github.com/user-attachments/files/17853705/log.txt)

### Expected Behavior

Steam millennium starts

### Steps To Reproduce

1. Run Arch Linux
2. Install steam through Pacman w/ multilib repo
3. install steam milliennium via `curl -fsSL "https://raw.githubusercontent.com/SteamClientHomebrew/Millennium/main/scripts/install.sh" | sh`
4. Run start.sh in konsole
5. Observe

### Operating System

Linux

### Anything else?

System Information via FastFetch

> [plato@dave ~]$ fastfetch -l none
plato@dave
OS: Arch Linux x86_64
Host: X570 AORUS MASTER (-CF)
Kernel: Linux 6.11.7-arch1-1
Uptime: 1 hour, 46 mins
Packages: 999 (pacman), 33 (flatpak)
Shell: bash 5.2.37
Display (DELD10F): 2560x1440 @ 165 Hz in 31" [External] *
Display (DELL 2009W): 1680x1050 @ 60 Hz in 20" [External]
Display (DELL P2012H): 1600x900 @ 60 Hz in 20" [External]
DE: KDE Plasma 6.2.3
WM: KWin (Wayland)
WM Theme: Breeze
Theme: Breeze (UtterlySweet) [Qt], Breeze-Dark [GTK2], Breeze [GTK3]
Icons: BeautySolar [Qt], BeautySolar [GTK2/3/4]
Font: Noto Sans (10pt) [Qt], Noto Sans (10pt) [GTK2/3/4]
Cursor: Sweet (24px)
Terminal: konsole 24.8.3
CPU: AMD Ryzen 7 5800X3D (16) @ 4.55 GHz
GPU: NVIDIA GeForce RTX 3080 Ti
Memory: 5.48 GiB / 31.27 GiB (18%)
Swap: 946.67 MiB / 4.00 GiB (23%)
Disk (/): 91.60 GiB / 317.03 GiB (29%) - ext4
Disk (/home/plato/drives/Backup): 202.69 GiB / 1.82 TiB (11%) - fuseblk [Read-only]
Disk (/home/plato/drives/FGames): 10.36 GiB / 457.38 GiB (2%) - ext4
Disk (/home/plato/drives/Games): 306.82 GiB / 915.82 GiB (34%) - ext4
Disk (/home/plato/drives/Shenanigans): 74.06 GiB / 698.64 GiB (11%) - fuseblk [Read-only]
Disk (/home/plato/drives/Windows): 345.72 GiB / 604.91 GiB (57%) - fuseblk [Read-only]
Local IP (enp5s0): 192.168.1.49/24
Locale: en_US.UTF-8

**Comment by shdwmtr** [comment]
I assume it happens every launch?

**Comment by agedplato** [comment]
yep! every launch it does this exact thing. I have tried reinstalling steam and millennium. not sure what else to try and troubleshoot.

**Comment by shdwmtr** [comment]
It could possibly have something to do with a firewall or other similar network blocking software.

**Comment by agedplato** [comment]
Only thing I'm running in that area is DNSmasq, otherwise I don't have a firewall setup currently. Nothing on my router I can think of either.

**Comment by agedplato** [comment]
After completely wiping my system and reinstalling the problem persists. Going to try a VM next to see what happens, will try and dual boot my laptop later to get some more logs.

**Comment by Bread-Ch4n** [comment]
You tried starting millennium with ``~/.millennium/start.sh -cef-enable-debugging -dev``?

Also idk if these 2 lines suppost to be there:
```
STEAM Could not find platform independent libraries <prefix>
STEAM Could not find platform dependent libraries <exec_prefix>
```

**Comment by shdwmtr** [comment]
@agedplato Weird, I wish I could provide more insight but I really don't have much to go off of, and even with a deeper explanation I'd likely still not know.

**Comment by agedplato** [comment]
Ill keep messing around with it and see what i can find. If i figure out what is causing the issue ill be sure to add it to the thread. Thanks for trying at the very least!

**Comment by shdwmtr** [comment]
Should have been fixed in an update a while ago, lmk if not.
